<ui version="4.0" >
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>482</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>3</hsizetype>
    <vsizetype>3</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab" >
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Miscellaneous</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="saveWindowSizeCheckBox" >
            <property name="text" >
             <string>Save current window size and position when exiting</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Logging</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QCheckBox" name="logToFileCheckBox" >
              <property name="text" >
               <string>Log to file</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType" >
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>20</width>
                  <height>10</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="fileLogDetailLabel" >
                <property name="sizePolicy" >
                 <sizepolicy>
                  <hsizetype>5</hsizetype>
                  <vsizetype>1</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize" >
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text" >
                 <string>Level of detail</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="fileLogDetailComboBox" >
                <property name="sizePolicy" >
                 <sizepolicy>
                  <hsizetype>5</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize" >
                 <size>
                  <width>220</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="currentIndex" >
                 <number>1</number>
                </property>
                <item>
                 <property name="text" >
                  <string>Debugging</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>Configuration (recomended)</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>General information</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>Warnings</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>Only severe events</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>40</width>
                  <height>10</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType" >
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>20</width>
                  <height>10</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="fileLogFilenameLineEdit" />
              </item>
              <item>
               <widget class="QPushButton" name="fileLogBrowseButton" >
                <property name="text" >
                 <string>Browse...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>10</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="logToConsoleCheckBox" >
              <property name="text" >
               <string>Log to console</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType" >
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>20</width>
                  <height>10</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="consoleLogDetailLabel" >
                <property name="sizePolicy" >
                 <sizepolicy>
                  <hsizetype>5</hsizetype>
                  <vsizetype>1</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize" >
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text" >
                 <string>Level of detail</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="consoleLogDetailComboBox" >
                <property name="sizePolicy" >
                 <sizepolicy>
                  <hsizetype>5</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize" >
                 <size>
                  <width>220</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="currentIndex" >
                 <number>1</number>
                </property>
                <item>
                 <property name="text" >
                  <string>Debugging</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>Configuration (recomended)</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>General information</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>Warnings</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>Only severe events</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>40</width>
                  <height>10</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>10</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>Logging changes will take effect at next invocation</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pluginsTab" >
      <attribute name="title" >
       <string>Plugins</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
